meinberlin/apps/budgeting/templates/meinberlin_budgeting/proposal_create_form.html

```diff
@@ -2,6 +2,23 @@
 {% load i18n %}
 
 {% block title %}{% translate 'Create a new proposal' %} &mdash; {{ block.super }}{% endblock %}
+{% block breadcrumbs %}
+    <div id="layout-grid__area--contentheader">
+        <div id="content-header">
+            <nav class="breadcrumb" aria-label="{% translate 'You are here:' %}">
+                <ol>
+                    {% url 'meinberlin_plans:plan-list' as project_list %}
+                    <li><a href="{% url 'meinberlin_plans:plan-list' %}">{% trans 'Project Overview' %}</a></li>
+                    <li><a href="{{ project.get_url }}">{% if project.name|truncatechars:50 %}{{ project.name|truncatechars:50 }}{% else %}{{ project.name }}{% endif %}</a></li>
+                    {% if module.is_in_module_cluster  %}
+                        <li><a href="{{ module.get_detail_url }}">{% if module.name|truncatechars:50 %}{{ module.name|truncatechars:50 }}{% else %}{{ module.name }}{% endif %}</a></li>
+                    {% endif %}
+                    <li class=”active” aria-current="page">{% translate 'Submit a new proposal for this project' %}</li>
+                </ol>
+            </nav>
+        </div>
+    </div>
+{% endblock %}
```

Comment:
@m4ra I ended up just adding the breadcrumb code to all required templates as it's different for each page but i'm not sure if this is the best django/pythony way to do it?